app/controllers/idv/cancellations_controller.rb

```diff
@@ -36,16 +36,6 @@ def destroy
       end
     end
 
-    def exit
```

Comment:
I didn't see when this was added, but I'm glad we're removing. I think `exit` is a bad name for an action because it shadows the `Kernel#exit` method built into every Ruby object, the one that exits the program, it would be a bad mistake for code to call `#exit` on the wrong object instead of this controller